adb_graphics/datahandler/gribdata.py

```diff
@@ -112,6 +112,17 @@ def field_diff(self, values, variable2, level2, **kwargs):
 
         return diff
 
+    def field_column_max(self, values, variable, **kwargs):
+
+        # pylint: disable=unused-argument
+
+        ''' Returns the column max of the values. '''
+
+        vals = self.values(name=variable, level='1000ft', one_lev=False)
+        maxvals = vals.max(axis=0)
+
```

Comment:
Is the selection of level='1000ft' arbitrary, since you set one_lev=False, or is there a particular reason to select this level?
I assume axis=0 is the vertical axis?

Reply:
yes, '1000ft' was arbitrary but has been removed.  Also, yes, axis=0 is the first array variable, in the RRFS grib2 files that's vertical.